.circleci/skip-e2e-pr-checks.sh

```diff
@@ -12,7 +12,9 @@ else
   exit 0
 fi
 
-# Exiting if the commit message contains 'skip e2e' string
+# Get the commit message. Exiting job early if the commit message contains 'skip e2e' string.
+COMMIT_MESSAGE=$(git log -1 --pretty=format:"%s")
```

Comment:
Get commit msg in shell script directly. No need to create a env. variable for it.